ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Block.hs

```diff
@@ -16,90 +16,37 @@ module Ouroboros.Consensus.Cardano.Block (
     -- checks from GHC. But GHC expects a data type, not a type family, that's
     -- why we sometimes mention the data type of the instance in these exports
     -- instead of the abstract type family.
-  , HardForkBlock (
-        BlockByron
-      , BlockShelley
-      , BlockAllegra
-      , BlockMary
-      )
+  , HardForkBlock (BlockByron, BlockShelley, BlockAllegra, BlockMary)
     -- * Headers
   , CardanoHeader
-  , Header (
```

Comment:
I'm of two minds about alphabetizing these. For the most part, I would expect such lists to be alphabetized. But the era names are the one example where the non-alphabetical ordering is intentional. (Though I don't know if that's ultimately the wisest choice.)

Reply:
Responded here https://github.com/input-output-hk/ouroboros-network/pull/3009#discussion_r599467748

Reply:
We opened upstream jaspervdj/stylish-haskell#360